contrib/install_db4.sh

```diff
@@ -64,7 +64,7 @@ http_get() {
 
 mkdir -p "${BDB_PREFIX}"
 http_get "${BDB_URL}" "${BDB_VERSION}.tar.gz" "${BDB_HASH}"
-tar -xzvf ${BDB_VERSION}.tar.gz -C "$BDB_PREFIX"
+tar -xzf ${BDB_VERSION}.tar.gz -C "$BDB_PREFIX"
```

Comment:
This seems unrelated to the actual change? I guess it's fine to remove the verbose option, I am just curious if this was done on purpose.

Reply:
@fjahr This change is intentional: this PR uses `contrib/install_db4.sh` and the `-v` flag fills the CI output with an unnecessary file listing. That said: if anyone finds a need for the verbose flag (`-v`) here I'm happy to revert :)

Reply:
Removed this change to minimize the diff and avoid review distractions :)